packages/components/src/FilterButton/_docs/filter-buttons.stickersheet.stories.tsx

```diff
@@ -79,6 +79,14 @@ const StickerSheetTemplate: StickerSheetStory = {
       </StickerSheet>
     </>
   ),
+  parameters: {
+    pseudo: {
+      hover: ".story__filter-button-base--hover",
+      active: ".story__filter-button-base--active",
+      focus: ".story__filter-button-base--focus",
+      focusVisible: ".story__filter-button-base--focus",
```

Comment:
Maybe I should just use `id`s. As long as we never render multiple stickersheets at the same time, there shouldn't be a conflict.